repositories/cartItem-repository.js

```diff
@@ -0,0 +1,27 @@
+const uuid = require('uuid');
+
+let cartItems = [
+    {
+        'cartItem_Id': uuid.v4(),
+        'cart_Id': uuid.v4(),
+        'quantity': 4
+    }
+];
+
+ const selectCartItems  = () => ({
+     rows: cartItems,
+     error: new Error(),
+     driver: 'postgres'
+ });
+
+ const selectCartItemByCartItemId = (cartItemId) =>
+     cartItems.find((cartItem) => cartItem['cartItem_Id'] === cartItemId);
+
+ const selectCartItemByCartId = (cartId) =>
+     cartItems.find((cartItem) => cartItem['cart_Id'] === cartId);
```

Comment:
```suggestion
     cartItems.filter((cartItem) => cartItem['cart_Id'] === cartId);
```

selectCartItemsByCartId, we want to fetch all cart items by a cartId